src/components/Drawer/Drawer.test.jsx

```diff
@@ -0,0 +1,113 @@
+import { composeStories } from "@storybook/testing-react";
+import { render, screen } from "@testing-library/react";
+import { axe } from "jest-axe";
+import { vi } from "vitest";
+
+import { Drawer } from ".";
+import * as DrawerStories from "./Drawer.stories";
+
+const { Default, Templated } = composeStories(DrawerStories);
+
+describe("Drawer", () => {
+	it("fully renders without exploding", () => {
+		render(<Drawer aria-label="example drawer">content</Drawer>);
+
+		const rootElement = screen.getByRole("dialog");
+		expect(rootElement).toBeInTheDocument();
+	});
+
+	it("fully renders with a title without exploding", () => {
+		const { getByRole } = render(<Drawer title="example title" />);
+
+		const rootElement = getByRole("dialog");
+		expect(rootElement).toBeTruthy();
+	});
+
+	it("fully renders with a JSX title without exploding", () => {
+		const { getByRole } = render(
+			<Drawer title={<span>example JSX title</span>} />,
+		);
+
+		const rootElement = getByRole("dialog");
+		expect(rootElement).toBeTruthy();
+	});
+
+	it("allows the passing of `<div>` props", () => {
```

Comment:
I think that you're lying by accident here. Looking at your code, it appears to be extending the dialog props. Thus, I recommend updating to follow typescript lingo and be accurate: 
```suggestion
	it("extends `<dialog>` properties", () => {
```

Reply:
done